homeassistant/components/elmax/__init__.py

```diff
@@ -42,6 +43,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     # Store a global reference to the coordinator for later use
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
 
+    # Add the cover feature only if supported by the current panel.
+    if coordinator.data is not None and coordinator.data.cover_feature:
+        ELMAX_PLATFORMS.append(Platform.COVER)
```

Comment:
Move this to the cover platform itself. Let the cover platform decide if it adds entities or not (in the platform setup).

Reply:
Done, addressed in 182a67378bfda2e4eae61cb04f3a3147ae2dc98e